jsweaver/src/pt/up/fe/specs/jackdaw/abstracts/weaver/AJackdawWeaver.java

```diff
@@ -8,8 +8,8 @@
 
 /**
  * Abstract Weaver Implementation for JackdawWeaver<br>
- * Since the generated abstract classes are always overwritten, their implementation should be done by extending those abstract classes with user-defined classes.<br>
- * The abstract class {@link pt.up.fe.specs.jackdaw.abstracts.AJackdawWeaverJoinPoint} can be used to add user-defined methods and fields which the user intends to add for all join points and are not intended to be used in LARA aspects.
+Since the generated abstract classes are always overwritten, their implementation should be done by extending those abstract classes with user-defined classes.<br>
+The abstract class {@link pt.up.fe.specs.jackdaw.abstracts.AJackdawWeaverJoinPoint} can be used to add user-defined methods and fields which the user intends to add for all join points and are not intended to be used in LARA aspects.
```

Comment:
What happened here?

Reply:
If the question is about the change, it appears to be related to auto-formatting.

About the contents, apparently this message was written before the same class was used to host attributes and actions that are common to all join points, and that can and should be used in LARA aspects. This is outdated, the weaver generator in lara-framework will be updated to change the message so that it better reflects how it is used nowadays.